src/main/java/org/jenkinsci/plugins/configfiles/buildwrapper/ManagedFileUtil.java

```diff
@@ -97,12 +105,12 @@ public FilePath call() throws IOException {
 
             FilePath target = null;
             if (createTempFile) {
-                target = ManagedFileUtil.createTempFile(build.getWorkspace().getChannel());
+                target = ManagedFileUtil.createTempFile(ws.getChannel());
             } else {
                 
                 String expandedTargetLocation = managedFile.targetLocation;
                 try {
-                    expandedTargetLocation = TokenMacro.expandAll(build, listener, managedFile.targetLocation);
+                    expandedTargetLocation = TokenMacro.expandAll((AbstractBuild<?, ?>) build, listener, managedFile.targetLocation);
```

Comment:
I think you could just skip the `token-macro` calls if you do not have an
`AbstractBuild`. It is useless from a workflow anyway, since any variable
substitutions would have been done in Groovy before the step is even
`perform`ed.